### PR TITLE
test: add tests to VisualPicker component

### PR DESCRIPTION
**Pull request by wildergd**
fix: #861 

Changes proposed in this PR:
- add tests to VisualPicker component

[ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/90milesbridge/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

@90milesbridge/tigger

### CODE REVIEW

File: src/components/VisualPickerOption/__test__/visualPickerOption.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        const component = mount(<VisualPickerOption name="option1" groupName={groupName} />);
+        expect(component.find('input').prop('name')).toBe(groupName);
+    });
+    it('should mark the input element as checked', () => {
+        const value1 = 'option1';
+        const value2 = ['option1'];
+        const component1 = mount(<VisualPickerOption name="option1" value={value1} />);
+        const component2 = mount(<VisualPickerOption name="option1" multiple value={value2} />);
+        expect(component1.find('input').prop('checked')).toBe(true);
+        expect(component2.find('input').prop('checked')).toBe(true);
```
Comment (LeandroTorresSicilia) [inline src/components/VisualPickerOption/__test__/visualPickerOption.spec.js]: let's split this in tow tests:
- should mark the input element as checked when is not multiple.
- should mark the input element as checked when is multiple
also add:
- should not mark the input element as checked when is multiple and value passed is not an array

File: src/components/VisualPicker/__test__/visualPicker.spec.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                <VisualPickerOption name="option2" />
+                <VisualPickerOption name="option3" />
+            </VisualPicker>,
+        );
+        component
+            .find('VisualPickerOption[name="option3"]')
+            .find('input[type="checkbox"]')
+            .simulate('change', { target: { checked: true } });
+        expect(onChangeFn).toHaveBeenCalledWith(['option1', 'option3']);
+    });
```
Comment (LeandroTorresSicilia) [inline src/components/VisualPicker/__test__/visualPicker.spec.js]: add tests for other cases when onChange:
. should call onChange with the new option selected when have a one selected initially (this one for single selection).
. should call onChange with the right options when is multiple and the initial value is not an array.
. should call onChange with the right options when is multiple, have values selected initially and deselect one.

File: src/components/VisualPickerOption/__test__/visualPickerOption.spec.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        const component = mount(<VisualPickerOption name="option1" multiple />);
+        expect(component.find('input').prop('type')).toBe('checkbox');
+    });
+    it('should set correct name to the input element', () => {
+        const groupName = 'group-1';
+        const component = mount(<VisualPickerOption name="option1" groupName={groupName} />);
+        expect(component.find('input').prop('name')).toBe(groupName);
+    });
+    it('should mark the input element as checked', () => {
+        const value1 = 'option1';
```
Comment (LeandroTorresSicilia) [inline src/components/VisualPickerOption/__test__/visualPickerOption.spec.js]: you need to remove this test

Comment (wildergd) [reply src/components/VisualPickerOption/__test__/visualPickerOption.spec.js]: I forgot to remove it

**Pull request merged**